fiftyone/migrations/revisions/v0_6_2.py

```diff
@@ -8,6 +8,20 @@
 import pymongo as pm
 
 
+def _up_convert_polyline_points(d):
+    cls = d.get("cls", None)
+    if cls == "Polyline":
+        if "points" not in d:
+            return False
+        d["points"] = [d["points"]]
```

Comment:
Is the current type of `d["points"]` checked somewhere (or does it not need to be)?

Reply:
I don't think it needs to be. Mongengine validates these fields.